src/components/chat-message.tsx

```diff
@@ -26,55 +20,59 @@ export interface ChatMessageProps {
 		time: string
 	) => void;
 	isUnauthenticated?: boolean;
-	failedMessage?: boolean;
 }
 
 export const ChatMessage = ( {
-	children,
 	id,
-	messageId,
-	isUser,
+	message,
 	className,
 	siteId,
-	blocks,
 	updateMessage,
+	children,
 	isUnauthenticated,
-	failedMessage,
 }: ChatMessageProps ) => {
 	return (
 		<div
+			key="container"
```

Comment:
Out of curiosity, what's the purpose of adding the keys `container` and `content`?

Reply:
They were there, because of `<AnimatePresense/>` component. However, as this now is moved up, they are no longer needed. Removed! Thanks!

Reply:
fixed